tsl/test/expected/gapfill.out

```diff
@@ -1080,3 +1070,338 @@ FROM metrics_tstz m1 WHERE device_id=1 GROUP BY 1,2;
  Mon Jan 01 08:00:00 2018 PST |         1 |          -5 |    
 (4 rows)
 
+\set ON_ERROR_STOP 0
+-- bucket_width non simple expression
+SELECT
+  time_bucket_gapfill(t,t)
+FROM (VALUES (1),(2)) v(t)
+WHERE true AND true
+GROUP BY 1;
+ERROR:  invalid time_bucket_gapfill argument: bucket_width must be a simple expression
```

Comment:
This could also use a `could not deduce start boundary from WHERE clause`, right now it'll just lead to questions "why's it complaining about `bucket_width` when I didn't use it?".

Reply:
bucket_width is checked before start/finish and this constraint is always true, bucket_width always needs to be simple expression independant of start/finish

Reply:
this was not checked earlier but that was on oversight